packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -657,11 +657,11 @@ export class FluidDataStoreRuntime extends EventEmitter implements IFluidDataSto
         // Get the object snapshot only if the data store is Bound and its graph is attached too,
```

Comment:
I'd love to see how FluidDataStoreRuntime.summarize() and getAttachSnapshot() APIs would change as well in same PR, even if we do not use results yet in any form on container runtime.
I've left a comment that signature of APIs across layers should be the same, we should use this opportunity to move away from snapshots and push deeper (hide it further) legacy structures.

Reply:
Updated FluidDataStoreRuntime.summarize() to return the routes. I haven't changed getAttachSnapshot() here because it is updated in my other PR (#4230) which removes snapshot. I will update whichever PR goes in later.